changelog.d/19785_https_svcb_record_types_support.enhancement.md

```diff
@@ -0,0 +1 @@
+Added support for parsing HTTPS (type 65) and SVCB (type 64) resource records from DNS messages
```

Comment:
```suggestion
Added support for parsing HTTPS (type 65) and SVCB (type 64) resource records from DNS messages
authors: @esensar
```

In Vector there is a contributors field that is rendered on the website for the release notes, if you'd like attribution.

Reply:
Oh, cool, thanks!